### DISCUSSION TITLE
Window decorations issue

### DISCUSSION THREAD

**Discussion by Ewanse** [emilk/egui#4316] [Q&A]
Hi! 
When I use `with_decorations(false)` there is some black space in top and left of the window.
Is there any way to remove this black space?
![Screenshot_1](https://github.com/emilk/egui/assets/147420354/d1b28574-f6d7-4faa-a988-c9384acda419)

**Comment by YgorSouza** [comment]
Maybe your UI code has some unwanted padding, or maybe it is only a matter of making the viewport [transparent](https://docs.rs/egui/latest/egui/viewport/struct.ViewportBuilder.html#method.with_transparent). Check the [custom_window_frame](https://github.com/emilk/egui/tree/2342788973120552c012f028bfda576fea87738b/examples/custom_window_frame) example.

**Reply by Ewanse** [reply]
This happens with all the examples I tried.
custom_window_frame example:
![image](https://github.com/emilk/egui/assets/328723/7c283f32-a878-4a9a-8876-c75d00ae4083)

**Reply by YgorSouza** [reply]
So the window is cropped? Yeah that's odd. This example works fine for me, both on Windows 11 and on Fedora 39. If it's not working for you, you should open an issue with your specs (OS version, CPU, GPU etc) to see if we can figure out what is wrong.

**Reply by steelx** [reply]
worked fine. Im using eframe template. (just that now I cannot move the window anywhere)
```rs
let native_options = eframe::NativeOptions {
        viewport: egui::ViewportBuilder::default()
            .with_decorations(false)
```

![image](https://github.com/emilk/egui/assets/3184210/a9502da8-8edb-42e0-b341-575766a87340)
